src/potemkin/collections.clj

```diff
@@ -118,8 +118,10 @@
   Object
   (hashCode [this]
     (reduce
-      (fn [acc [k v]]
-        (unchecked-add acc (bit-xor (.hashCode k) (.hashCode v))))
+     (fn [acc [k v]]
+       (unchecked-add acc (bit-xor
+                           (if k (.hashCode k) 0)
```

Comment:
and that's not gonna be a nitpick but we just won't hash false values with my code :)